src/index.html

```diff
@@ -17,16 +17,23 @@
     />
   </head>
   <body>
-    <input
-      type="text"
+    <form
       data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+      class="form"
+    >
+      <input
+        class="form__search-big"
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
The input for the big search bar is correctly implemented with the `data-qa="big"` attribute.